app_dart/lib/src/model/proto/internal/scheduler.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto2";
+
+message SchedulerConfig {
+    // Targets to run from this config.
+    repeated Target targets = 1;
+    // Git branches to run these targets against.
+    repeated string enabled_branches = 2;
+}
+
+message Target {
```

Comment:
Do we need a flaky flag somewhere?

Reply:
From feedback on go/flutter-build-test-infra, we're renaming it as bringup